src/redux/ui/screens/data/actions/transactions/index.ts

```diff
@@ -0,0 +1,63 @@
+import { cloneDeep } from 'lodash';
+import { Dispatch } from 'react';
+import { IHandleCtaActionContext } from '../index';
+import { Contracts } from '../../../../../../core/blockchain/zilliqa/config';
+import { TransactionStatus } from '../../../../../../core/wallet/types';
+import { IReduxState } from '../../../../../state';
+import { captureException as SentryCaptureException } from '@sentry/react-native';
+import { IAction } from '../../../../../types';
+import {
+    openProcessTransactions,
+    setProcessTransactions
+} from '../../../../process-transactions/actions';
+import { buildContractCallTransaction } from './transaction-builder';
+
+export * from './transaction-builder';
+
+export interface IContractCallParams {
+    rawData: any;
+    blockNumber: number;
+    tokenSymbol: string;
```

Comment:
I think this one can also be omitted, usually we know in what token the fees are paid